### PR TITLE
Added support for MessagePack binding and rendering

### PR DESCRIPTION
**Pull request by harindup**

### CODE REVIEW

**Comment by dilshank** [comment]
+1

File: binding/msgpack.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2014 Manu Martinez-Almeida.  All rights reserved.
```
Comment (appleboy) [inline binding/msgpack.go]: 2014 -> 2017

File: render/msgpack.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Copyright 2014 Manu Martinez-Almeida.  All rights reserved.
```
Comment (appleboy) [inline render/msgpack.go]: 2014 -> 2017

File: binding/msgpack.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+// Use of this source code is governed by a MIT style
+// license that can be found in the LICENSE file.
+
+package binding
+
+import (
+	"github.com/ugorji/go/codec"
```
Comment (appleboy) [inline binding/msgpack.go]: put external package to bottom and local page to top.

**Comment by appleboy** [comment]
LGTM

**Comment by appleboy** [comment]
@harindup Thanks

**Pull request merged**